test/jdk/java/lang/invoke/MethodHandleInvokeUOE.java

```diff
@@ -21,15 +21,16 @@
  * questions.
  */
 
-/* @test
+/*
+ * @test
+ * @bug 8343377
```

Comment:
I'm surprised adding `@bug` in this test.  As no change made to this test, I think no need to add the bug ID.

Reply:
Reverted these two tests.